arch/generic/crc32_braid_c.c

```
@@ -206,8 +211,15 @@ Z_INTERNAL uint32_t crc32_braid(uint32_t crc, const uint8_t *buf, size_t len) {
     return ~crc32_copy_small(crc, NULL, buf, len, (BRAID_N * BRAID_W) - 1, 0);
 }
 
+#endif /* CRC32_BRAID_FALLBACK || !WITHOUT_CHORBA */
+
+/* Public braid functions compiled when braid fallback or chorba (which depends on braid) is used */
+#if defined(CRC32_BRAID_FALLBACK) || !defined(WITHOUT_CHORBA)
+
 Z_INTERNAL uint32_t crc32_copy_braid(uint32_t crc, uint8_t *dst, const uint8_t *src, size_t len) {
```

Cppcheck (2.19.0)

[style] The function 'crc32_copy_braid' is never used.

(unusedFunction)